extension_explorer/templates/home.html

```diff
@@ -14,7 +14,7 @@ <h1 class="display-4">
         </h1>
 
         <p class="lead">
-          {{ _('Disclose more data about your contracting processes, using extensions to the <a href="https://standard.open-contracting.org/latest/%(lang)s/">Open Contracting Data Standard</a>.', lang=lang) }}
+          {{ _('Find extensions to the <a href="https://standard.open-contracting.org/latest/%(lang)s/">Open Contracting Data Standard</a>.', lang=lang) }}
```

Comment:
I would not make this change. It's important to include the ultimate goal here. Finding extensions is just a means to an end.

Reply:
I was aiming for wording that worked for publishers and users. However, I suppose the main audience if publishers, so happy to revert the change.

Reply:
Done in https://github.com/open-contracting/extension-explorer/pull/67/commits/f254e86f77cf25627685c5dfadf3d5db662db0c7